### DISCUSSION TITLE
Trying to use MaterialUI but getting DeprecationWarning

### DISCUSSION THREAD

**Discussion by techwithhuz** [idom-team/idom#1108] [Question] [answered]
While using materialUI getting below warning

**DeprecationWarning: module_from_template() is deprecated due to instability - use the Javascript Components API instead.**

But couldn't find document referencing how to use JavaScript components API.

Is there any document link for reference?

**Comment by rmorshea** [comment, marked as answer]
We're working on a [better way](https://github.com/reactive-python/reactpy/issues/1001) to integrate with Javascript, but for now, the docs on custom JS components can be found [here](https://reactpy.dev/docs/guides/escape-hatches/javascript-components.html#custom-javascript-components).
